erts/preloaded/src/socket.erl

```diff
@@ -2787,22 +2787,20 @@ enc_type(_, raw)       -> ?SOCKET_TYPE_RAW;
 enc_type(_, seqpacket) -> ?SOCKET_TYPE_SEQPACKET;
 enc_type(_, Type)      -> invalid_type(Type).
```

Comment:
`enc_type/2` could also be rewritten into arity 1 for symmetry, and the call to it in `open/4`